webui.py

```diff
@@ -39,7 +39,7 @@ def api_only():
 
     print(f"Startup time: {startup_timer.summary()}.")
     api.launch(
-        server_name="0.0.0.0" if cmd_opts.listen else "127.0.0.1",
+        server_name=cmd_opts.server_name or ("0.0.0.0" if cmd_opts.listen else "127.0.0.1"),

```

Comment:
As pointed out on Discord, this should use the `initialize_util.gradio_server_name()` function that's available in the `dev` branch.

The `dev` branch is what this PR should be targeting; a PR targeting `master` will not get merged.